pkg/controllers/disruption/orchestration/queue.go

```diff
@@ -181,24 +193,19 @@ func NewCommand(replacements []string, candidates []*state.StateNode, id types.U
 func (q *Queue) Register(_ context.Context, m manager.Manager) error {
 	return controllerruntime.NewControllerManagedBy(m).
 		Named("disruption.queue").
-		WatchesRawSource(singleton.Source()).
-		Complete(singleton.AsReconciler(q))
+		WatchesRawSource(source.Channel(q.source, &handler.TypedEnqueueRequestForObject[*v1.NodeClaim]{})).
+		WithOptions(controller.Options{
+			RateLimiter:             reasonable.RateLimiter(),
```

Comment:
What's the max backoff that can happen here? Should we make this more similar to the eviction queue because I think that we should think about using a consistent poll and not have such a long maxBackoff just because we aren't hitting external APIs and are only hitting the apiserver